Interfaces/scripts/dirac-admin-get-site-mask.py

```diff
@@ -25,7 +25,7 @@
 
 gLogger.setLevel('ALWAYS')
 
-result = diracAdmin.getSiteMask(printOutput=True)
+result = diracAdmin.getSiteMask(printOutput=True, status="Active")
```

Comment:
The method is `def getSiteMask(self, printOutput=False, status='Active'):` so active is anyway the defaul.